tmail-backend/mailbox/encrypted/cassandra/src/main/java/com/linagora/tmail/encrypted/cassandra/table/CassandraEncryptedEmailStoreModule.scala

```diff
@@ -0,0 +1,36 @@
+package com.linagora.tmail.encrypted.cassandra.table
+
+import com.datastax.driver.core.DataType.{cboolean, cint, text, uuid}
+import com.datastax.driver.core.schemabuilder.{Create, SchemaBuilder}
+import com.datastax.driver.core.schemabuilder.Create.Options
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.KeyCaching.ALL
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.rows
+import org.apache.james.backends.cassandra.components.CassandraModule
+import org.apache.james.backends.cassandra.utils.CassandraConstants.DEFAULT_CACHED_ROW_PER_PARTITION
+
+object CassandraEncryptedEmailStoreModule {
+  val EMAIL_CONTENT_MODULE: CassandraModule = CassandraModule.table(EncryptedEmailTable.TABLE_NAME)
+    .comment("Holds content of encrypted email")
+    .options((options: Options) => options
+      .caching(ALL, rows(DEFAULT_CACHED_ROW_PER_PARTITION))
+      .compressionOptions(SchemaBuilder.lz4.withChunkLengthInKb(8)))
+    .statement((statement: Create) => statement
+      .addPartitionKey(EncryptedEmailTable.MESSAGE_ID, uuid)
+      .addColumn(EncryptedEmailTable.ENCRYPTED_PREVIEW, text())
+      .addColumn(EncryptedEmailTable.ENCRYPTED_HTML, text())
+      .addColumn(EncryptedEmailTable.HAS_ATTACHMENT, cboolean())
+      .addColumn(EncryptedEmailTable.ENCRYPTED_ATTACHMENT_METADATA, text()))
+    .build
+
+  val MESSAGE_ID_BLOB_ID_MODULE: CassandraModule = CassandraModule.table(EncryptedEmailIdBlobIdTable.TABLE_NAME)
+    .comment("Holds mapping among messageId, blobId and position")
+    .options((options: Options) => options
+      .caching(ALL, rows(DEFAULT_CACHED_ROW_PER_PARTITION))
+      .compressionOptions(SchemaBuilder.lz4.withChunkLengthInKb(8)))
+    .statement((statement: Create) => statement
+      .addPartitionKey(EncryptedEmailIdBlobIdTable.MESSAGE_ID, uuid)
+      .addColumn(EncryptedEmailIdBlobIdTable.BLOB_ID, text())
+      .addColumn(EncryptedEmailIdBlobIdTable.POSITION, cint()))
```

Comment:
What is this `Position` ?

Reply:
for `def retrieveAttachmentContent(messageId: MessageId, position: Int): BlobId` method

Reply:
Then it should be a clustering column if you wish to use it in select collumns.

BTW using a frozen UDT list you can put it in the table above I guess...

Reply:
I added a new column in the above table. 
```
.addColumn(EncryptedEmailTable.POSITION_BLOB_ID_MAPPING, frozenMap(cint(), text())))
```
So this table is unnecessary => removed